docs/blog/2020-03-06-gatsby-and-jsparty-podcast/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Gatsby + JS Party = 💜"
+date: 2020-03-06
+author: "Dustin Schau"
+image: "images/jsparty.png"
+imageTitle: JS Party + Gatsby Banner
+excerpt: "I recently had the pleasure of joining the JS Party podcast to talk about all things Gatsby. Check it out!"
+tags:
+  - podcast
+  - cloud
+---
+
+Recently I had the pleasure of joining the [JSParty][jsparty] podcast to discuss _all_ things Gatsby. Starting with the classic question of "What is Gatsby?" progressing to the value of rehydration, future deliverables of the open-source product, [Gatsby Cloud][cloud], and everything in between! Quite truly, all things Gatsby with the JS Party crew!
+
+Check out the below podcast for more detail on why Gatsby, why Gatsby Cloud, and our future plans for Gatsby, the product.
+
+## 🔉JS Party Podcast 🔉
+
+<audio
+  data-theme="day"
+  data-src="https://changelog.com/jsparty/117/embed"
+  src="https://cdn.changelog.com/uploads/jsparty/117/js-party-117.mp3"
+  preload="none"
+  className="changelog-episode"
+  controls
+></audio>
+<p>
+  <a href="https://changelog.com/jsparty/117">
+    JS Party 117: Catching up with Gatsby
+  </a>{" "}
+  – Listen on <a href="https://changelog.com/">Changelog.com</a>
+</p>
+<script async src="//cdn.changelog.com/embed.js"></script>
```

Comment:
We honestly may not have a good pattern for this w/ MDX, we may need a custom component.